code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -598,9 +598,33 @@
 	required_other = 1
 	required_container = /obj/item/slime_extract/rainbow
 
-/datum/chemical_reaction/slime/slimeRNG/on_reaction(datum/reagents/holder)
-	var/mob/living/simple_animal/slime/random/S = new (get_turf(holder.my_atom))
-	S.visible_message("<span class='danger'>Infused with plasma, the core begins to quiver and grow, and a new baby slime emerges from it!</span>")
+/datum/chemical_reaction/slime/slimeRNG/on_reaction(datum/reagents/holder, created_volume)
+	if(created_volume >= 5)
+		var/obj/item/grenade/clusterbuster/slime/S = new (get_turf(holder.my_atom))
+		S.visible_message("<span class='danger'>Infused with plasma, the core begins to expand uncontrollably!</span>")
+		S.icon_state = "[S.base_state]_active"
+		S.active = TRUE
+		addtimer(CALLBACK(S, /obj/item/grenade.proc/prime), rand(15,60))
+		qdel(holder.my_atom) //deleto
+	else
+		var/mob/living/simple_animal/slime/random/S = new (get_turf(holder.my_atom))
```

Comment:
get_turf(...) => holder.my_atom.loc

Reply:
holder.my_atom.drop_location()?

Reply:
no, because transform operations should use loc directly

Reply:
seems weird but ok